flashinfer/trace/solutions/fp4_quantize/native.py

```
@@ -0,0 +1,49 @@
+api = "flashinfer.quantization.fp4_quantization.fp4_quantize"
+backend = "flashinfer"
+inputs = ("input", "global_scale", "sf_vec_size")
+outputs = ("quantized", "scales")
+api_kwargs = {
+    "input": "input",
+    "global_scale": "global_scale",
+    "sf_vec_size": "sf_vec_size",
+}
+
+
+def run(input, global_scale, sf_vec_size):
```

Ruff (0.16.1)

[error] Function argument `input` is shadowing a Python builtin

(A002)